README.md

```diff
@@ -113,7 +134,7 @@ Key ID               Database             Role
 
 And then delete the key with id: `200219702370238976`:
 
-```sh-session
+```sh
```

Comment:
I just noticed a few lines below that the header #command doesn't work anymore, it's list-of-commands